lib/wait.go

```diff
@@ -23,10 +23,8 @@ func (c *ContainerServer) ContainerWait(container string) (int32, error) {
 		func() (bool, error) {
 			if !isStopped(c, ctr) {
 				return false, nil
-			} else { // nolint
-				return true, nil // nolint
-			} // nolint
-
+			}
+			return true, nil
```

Comment:
@baude I wasn't sure about this one above. Do you know what the main reason for the else construct is there?

Reply:
couldn't one just return isStopped(c, ctr), nil ?

Reply:
Yeah, that should be possible too. I will change it.